MAVProxy/tools/MAVExplorer.py

```diff
@@ -336,12 +337,16 @@ def load_graphs():
             if filename.lower().endswith('.xml'):
                 gfiles.append(os.path.join(dirname, filename))
 
+    # list of parameter files to be skipped when loading graph XML
+    paramfiles = ["ArduSub.xml", "ArduPlane.xml", "APMrover2.xml", "ArduCopter.xml", "AntennaTracker.xml", "Blimp.xml", "Rover.xml"]
+    # loop though files
     for file in gfiles:
         if not os.path.exists(file):
             continue
-        # skip parameter files.  They specify an encoding, and under
-        # Python3 this leads to a warning from etree
-        if os.path.basename(file) in ["ArduSub.xml", "ArduPlane.xml", "APMrover2.xml", "ArduCopter.xml", "AntennaTracker.xml", "Blimp.xml", "Rover.xml"]:
+        # skip parameter and log message files.  They specify an encoding,
+        # and under Python3 this leads to a warning from etree
+        basename = os.path.basename(file)
+        if (basename in paramfiles) or basename.startswith("LogMessages_"):
```

Comment:
Perhaps we should create a subdirectory instead of extending the misery?

Possibly mirroring the `LogMessages` name we use on the autotest server?

Reply:
Sure. Sounds sensible to use a sub-folder and keep the name instead of renaming to LogMessage_<name>.xml.
In that case should probably use a subfolder for param XMLs too (separate PR).
This would make filtering them out easier, as it can be done but folder, instead of individual file name.

Reply:
Changed to use LogMessages subfolder as proposed.